provider/cluster/service.go

```diff
@@ -40,7 +40,7 @@ type Service interface {
 }
 
 // NewService returns new Service instance
-func NewService(ctx context.Context, session session.Session, bus pubsub.Bus, client Client) (Service, error) {
+func NewService(ctx context.Context, session session.Session, bus pubsub.Bus, client Client, clusterExternalPortQuantity uint) (Service, error) {
 	log := session.Log().With("module", "provider-cluster", "cmp", "service")
 
 	config := config{}
```

Comment:
Are the CLI flags not affecting this config object?  I think we need to get rid of the `env.Parse` here and just pass this object into `NewService`

Reply:
What object are you referring to here?

Reply:
`config{}`.

we were using module-specific configuration via environment variables before instead of stringing the settings through from the command line.  it looks like things have changed and many of the settings are being set from the command line now, which is fine, but I think that some settings have not been migrated over.

* [`provider/config.go`](https://github.com/ovrclk/akash/blob/c0becaf676e20fd807a125776b55c3c90968c19a/provider/config.go#L5-L7)
* [`provider/cluster/config.go`](https://github.com/ovrclk/akash/blob/c0becaf676e20fd807a125776b55c3c90968c19a/provider/cluster/config.go#L6-L7)
* [`provider/cluster/kube/settings.go`](https://github.com/ovrclk/akash/blob/c0becaf676e20fd807a125776b55c3c90968c19a/provider/cluster/kube/settings.go#L14-L25)

Reply:
Alright, I'll have to look some more at this in order to see what is going on.